src/service/resources-impl.js

```diff
@@ -685,7 +685,7 @@ export class ResourcesImpl {
     this.prerenderSize_ = this.viewer_.getPrerenderSize();
 
     const firstPassAfterDocumentReady =
-      this.documentReady_ && this.firstPassAfterDocumentReady_;
+      this.documentReady_ && this.firstPassAfterDocumentReady_ && this.ampInitialized_;
```

Comment:
I suggested a temporary, lower-risk fix that sets `this.maybeChangeHeight_ = true` when `ampInitialized_` gets set to `true`. Could we do that just before the cut this week (and then go with this fix after)?

This way we can avoid adding a behavior change on top of ~4 weeks of AMP changes to the next push.

Reply:
Sure, I'll begin testing that now

Reply:
It works!